packages/ts-transform/src/utils/css-transform.tsx

```diff
@@ -3,6 +3,7 @@ import autoprefixer from 'autoprefixer';
 import cssnano from 'cssnano-preset-default';
 import nested from 'postcss-nested';
 import whitespace from 'postcss-normalize-whitespace';
+import parser from 'postcss-selector-parser';
```

Comment:
nit: can we rename this to `selectorParser`?

```suggestion
import selectorParser from 'postcss-selector-parser';
```